assets/app/view/game/map.rb

```diff
@@ -76,7 +76,7 @@ def render
               end
 
               tiles = step.upgradeable_tiles(current_entity, @tile_selector.hex)
-              all_upgrades = @game.all_potential_upgrades(@tile_selector.hex.tile)
+              all_upgrades = @game.all_potential_upgrades(@tile_selector.hex.tile, @tile_selector.hex)
```

Comment:
is this really necessary? doesnn't the tile already have access to it's own hex?

Reply:
<<Face palm>> You are right. Pushed a commit that removed that parameter.